packages/mui-material/src/PaginationItem/PaginationItem.js

```diff
@@ -151,115 +158,142 @@ const PaginationItemPage = styled(ButtonBase, {
+              color: (theme.vars || theme).palette[color].main,
+              border: `1px solid ${
+                theme.vars
+                  ? `rgba(${theme.vars.palette[color].mainChannel} / 0.5)`
+                  : alpha(theme.palette[color].main, 0.5)
+              }`,
+              backgroundColor: theme.vars
+                ? `rgba(${theme.vars.palette[color].mainChannel} / ${theme.vars.palette.action.activatedOpacity})`
+                : alpha(theme.palette[color].main, theme.palette.action.activatedOpacity),
+              '&:hover': {
+                backgroundColor: theme.vars
+                  ? `rgba(${theme.vars.palette[color].mainChannel} / calc(${theme.vars.palette.action.activatedOpacity} + ${theme.vars.palette.action.focusOpacity}))`
+                  : alpha(
+                      theme.palette[color].main,
+                      theme.palette.action.activatedOpacity + theme.palette.action.focusOpacity,
+                    ),
+                // Reset on touch devices, it doesn't add specificity
+                '@media (hover: none)': {
+                  backgroundColor: 'transparent',
+                },
+              },
+              [`&.${paginationItemClasses.focusVisible}`]: {
+                backgroundColor: theme.vars
+                  ? `rgba(${theme.vars.palette[color].mainChannel} / calc(${theme.vars.palette.action.activatedOpacity} + ${theme.vars.palette.action.focusOpacity}))`
+                  : alpha(
+                      theme.palette[color].main,
+                      theme.palette.action.activatedOpacity + theme.palette.action.focusOpacity,
+                    ),
+              },
+              [`&.${paginationItemClasses.disabled}`]: {
+                borderColor: (theme.vars || theme).palette.action.disabledBackground,
+                color: (theme.vars || theme).palette.action.disabled,
+              },
             },
           },
-          [`&.${paginationItemClasses.focusVisible}`]: {
-            backgroundColor: theme.vars
-              ? `rgba(${theme.vars.palette[ownerState.color].mainChannel} / calc(${
-                  theme.vars.palette.action.activatedOpacity
-                } + ${theme.vars.palette.action.focusOpacity}))`
-              : alpha(
-                  theme.palette[ownerState.color].main,
-                  theme.palette.action.activatedOpacity + theme.palette.action.focusOpacity,
-                ),
-          },
-        }),
-        [`&.${paginationItemClasses.disabled}`]: {
-          borderColor: (theme.vars || theme).palette.action.disabledBackground,
-          color: (theme.vars || theme).palette.action.disabled,
-        },
-      },
-    }),
+        })),
+    ],
   }),
 );
 
 const PaginationItemPageIcon = styled('div', {
   name: 'MuiPaginationItem',
   slot: 'Icon',
-  overridesResolver: (props, styles) => styles.icon,
-})(({ theme, ownerState }) => ({
+  overridesResolver: (_props, styles) => styles.icon,
```

Comment:
```suggestion
  overridesResolver: (props, styles) => styles.icon,
```

Let's keep it the same.

Reply:
My bad, wasn't meant to commit this.